tests/test_monitor_stage.py

```diff
@@ -93,25 +93,25 @@ def test_build_single(mock_morph_tqdm, config):
 
     mock_stream = mock.MagicMock()
     mock_segment = mock.MagicMock()
-    mock_segment.make_sink.return_value = mock_stream
+    mock_segment.make_node_full.return_value = mock_stream
     mock_input = mock.MagicMock()
 
     m = MonitorStage(config)
     m._build_single(mock_segment, mock_input)
     m.on_start()
 
-    mock_segment.make_sink.assert_called_once()
+    mock_segment.make_node_full.assert_called_once()
     mock_segment.make_edge.assert_called_once()
 
-    sink_on_error = mock_segment.make_sink.call_args.args[2]
-    sink_on_completed = mock_segment.make_sink.call_args.args[3]
+    # sink_on_error = mock_segment.make_node_full.call_args.args[2]
+    # sink_on_completed = mock_segment.make_sink.call_args.args[3]
 
-    # This is currenlty just a log stmt, just verify that its callable
-    sink_on_error(RuntimeError("unittest"))
+    # # This is currenlty just a log stmt, just verify that its callable
+    # sink_on_error(RuntimeError("unittest"))
 
-    # Verify we close tqdm propperly on complete
-    sink_on_completed()
-    mock_morph_tqdm.stop.assert_called_once()
+    # # Verify we close tqdm propperly on complete
+    # sink_on_completed()
+    # mock_morph_tqdm.stop.assert_called_once()
```

Comment:
should this be moved to its own test?

Reply:
@dagardner-nv Do you know what the equivalent check would be now that `MonitorStage` is a node instead of a sink?

Reply:
Updated the test.
@cwharris part of the reason why this test is a bit involved is to ensure we are getting test coverage of the two nested functions, and getting access to the `sink_on_completed` requires first getting access to `node_fn`